### DISCUSSION TITLE
The telegram message does not contain all the data.

### DISCUSSION THREAD

**Discussion by Foresto-rip** [metalmatze/alertmanager-bot#149] [Help/Support] [answered]
Hello!
I have a problem. When alert comes, not all information is in the message. (look at the screen)

![Screenshot_2019-09-24-21-17-07-342_org telegram messenger](https://user-images.githubusercontent.com/55759986/65538830-df47b480-df10-11e9-88c0-46c230b9477d.png)

Why it may be?

**Comment by metalmatze** [comment]
It depends on your alerts and whether the key-value-pair in that object is available.
The default template assumes there to be a `.Annotations.message`.

https://github.com/metalmatze/alertmanager-bot/blob/a2db61819ddc5ab408a130e892f29fd02976a516/default.tmpl

You might need to change either the alerting rule or the template to match.

**Comment by Foresto-rip** [comment]
Ok. I changed this value, it works. But duration is left with the value - . May be you know why?
Thank you.

**Comment by swift201905** [comment]
> 好。我更改了此值，它起作用了。但是持续时间留有值-。也许你知道为什么吗？
> 谢谢。

how to setting   value ,can  show example，please

**Comment by runningman84** [comment]
This PR tries to address the missing message
https://github.com/metalmatze/alertmanager-bot/pull/113

**Comment by Georgii-M** [comment]
Hi, problem does not resolve
Tried to place default template. Alertmanager did not start with error "no such function /duration/ and /since/" Tried to remove them. Alertmanager loaded sucessfully, but still blank messages to Telegram. Tried to change 
.Annotations.{message|summary|description} to .CommonAnnotations no effect. Alertmanager config:
`webhook_configs:
   - send_resolved: true
     url: 'http://tgbot:8080'`
Messages are arriving to telegram like this, but Slack is good.
![image](https://user-images.githubusercontent.com/43551493/90648993-a06ecb80-e253-11ea-90ce-3b80a9ceae38.png)

**Comment by metalmatze** [comment, marked as answer]
You need to change the template of the alertmanager-bot not in alertmanager itself :)

**Comment by Georgii-M** [comment]
Thank you! Works perfectly! :))))

**Comment by esin** [comment]
@Georgii-M hey! did you solve problem?
